src/gui/accountsettings.cpp

```diff
@@ -471,42 +472,40 @@ void AccountSettings::slotAccountStateChanged()
 
     switch (state) {
     case AccountState::PausedDueToMetered:
-        showConnectionLabel(tr("Sync to %1 is paused due to metered internet connection.").arg(server));
```

Comment:
Please remove the `server` declaration as its now unused.
We can probably also remove the safeUrl?

Reply:
done